pages/support/[category]/index.js

```diff
@@ -91,6 +91,12 @@ const Category = ({ supportServices, query: { category } }) => {
   return (
     <>
       <CategoryStyled>
+        <button
```

Comment:
This button replaces the one in the navbar 
![Uploading Screenshot 2021-01-21 at 13.56.38.png…]()